### DISCUSSION TITLE
Why does inherits works for builtin profiles but not mine?

### DISCUSSION THREAD

**Discussion by LeLocTai** [OrcaSlicer/OrcaSlicer#4353] [General]
When setting up the slicer, one of the default process look like this:
```
{
    "type": "process",
    "setting_id": "GP004",
    "name": "0.08mm Extra Fine @MyKlipper",
    "from": "system",
    "instantiation": "true",
    "inherits": "fdm_process_klipper_common",
    "layer_height": "0.08",
    "bottom_shell_layers": "7",
    "top_shell_layers": "9"
}
```
In the slicer, the inherited values from fdm_process_klipper_common show up correctly.

However, when I tried to create my own profile in a similar structure, the settings that isn't in the child profile doesn't inherit from the parent, but instead some random values was used, probably default.
Theses are my profiles:
 - Parent: [Fast 0.4mm.json](https://github.com/SoftFever/OrcaSlicer/files/14507968/Fast.0.4mm.json)
 - Child: [Strong 0.6mm.json](https://github.com/SoftFever/OrcaSlicer/files/14507967/Strong.0.6mm.json)

**Comment by WizardUli** [comment]
Same problem here. I would very much like to use inherits on my own profiles and printer and filament definitions.
